src/nvim/screen.c

```diff
@@ -6450,7 +6450,6 @@ int screen_ins_lines (
   // - there is no valid screen
   // - the screen has to be redrawn completely
   // - the line count is less than one
```

Comment:
this comment doesn't match the condition. And the entire comment is kind of redundant and/or outdated at this point.